### ISSUE TITLE
Check into why some figure images do not always display on TEMPO ATBD

### ISSUE THREAD

**Issue by bwbaker1** [NASA-IMPACT/nasa-apt#896]
The TEMPO team is rather continuously encounter figure images not displaying on the generate PDF. Sometimes the captions are missing, but sometimes they are not (just related to images that don't appear. 

[TEMPO Nitrogen Dioxide Algorithm Theoretical Basis Document](https://www.earthdata.nasa.gov/apt/documents/tempo-nitrogen-dioxide/v1.0#doc-header)

Access can be granted for those with a production account (if needed). @wrynearson noticed a caching issue the other day, but doesn't seem to help now.

### Acceptance Criteria
- determine what is causing the issue

**Comment by wrynearson** [comment]
@jjfrench can you think of any reason why this would be happening? cc @frozenhelium.

I have not been able to reproduce this issue – the figures I've tested all download as expected for me.

**Comment by jjfrench** [comment]
Not entirely sure how these processes work but can we confirm it's a display issue or is the PDF not being generated properly? I can check for logs if I know what to look for (if it's a backend issue)

**Comment by wrynearson** [comment]
Yes @jjfrench, it's a PDF generation and backend issue. The PDFs are generated using Playwright (see https://github.com/NASA-IMPACT/nasa-apt/issues/622 and referenced issues/PRs for details).

It's strange because the generation is not happening on the user's side, but in the backend. Also, if a document hasn't been changed, we serve a cached version of the generated PDF. I don't understand why document PDFs that @bwbaker1 downloads would be different than the ones that I download, but they appear to be so.
